Binary files ru-old/added.bin and ru-new/added.bin differ
diff -ruN ru-old/added.txt ru-new/added.txt
--- ru-old/added.txt	1970-01-01 00:00:00.000000000 +0000
+++ ru-new/added.txt	2026-01-01 00:00:00.000000000 +0000
@@ -0,0 +1,3 @@
+line 1
+line 2
+line 3
Binary files ru-old/gone.bin and ru-new/gone.bin differ
Binary files ru-old/i.bin and ru-new/i.bin differ
diff -ruN ru-old/t.txt ru-new/t.txt
--- ru-old/t.txt	2026-01-01 00:00:00.000000000 +0000
+++ ru-new/t.txt	2026-01-01 00:00:00.000000000 +0000
@@ -1,4 +1,4 @@
 line 1
-line 2
+line two
 line 3
 line 4
